machine.go

```diff
@@ -50,7 +51,7 @@ type Machine struct {
 
 // Create a new machine object
 func NewMachine() (m *Machine) {
-	m = &Machine{memory: 2048, numcpus: runtime.NumCPU()}
+	m = &Machine{memory: 2048, numcpus: runtime.NumCPU(), showBoot: true}
```

Comment:
let showboot default to false

Reply:
I was going for minimal changes, but I'd personally prefer the default to be false, so no objection to changing that :-)

Reply:
The default is now not showing boot messages.